packages/frontapp/src/app/components/ListOfGenres/index.ts

```diff
@@ -0,0 +1,74 @@
+export const listOfGenres = [
+  {
+    id: 1,
+    value: 'Science Fiction',
+  },
+  {
+    id: 2,
+    value: 'Fantasy',
+  },
+  {
+    id: 3,
+    value: 'Comics & Graphic Novels',
+  },
+  {
+    id: 4,
+    value: 'Classics',
+  },
+  {
+    id: 5,
+    value: 'Development',
+  },
+  {
+    id: 6,
+    value: 'Biography',
+  },
+  {
+    id: 7,
+    value: 'Busines',
+  },
+  {
+    id: 8,
+    value: "Children's",
+  },
+  {
+    id: 9,
+    value: 'Crime',
+  },
+  {
+    id: 10,
+    value: 'History',
+  },
+  {
+    id: 11,
+    value: 'Horror',
+  },
+  {
+    id: 12,
+    value: 'Poetry',
+  },
+  {
+    id: 13,
+    value: 'Psychology',
+  },
+  {
+    id: 14,
+    value: 'Philosophy',
+  },
+  {
+    id: 15,
+    value: 'Religion',
+  },
+  {
+    id: 16,
+    value: 'Romance',
+  },
+  {
+    id: 17,
+    value: 'Art',
+  },
+  {
+    id: 18,
+    value: 'Other',
+  },
+];
```

Comment:
this is not a component

Reply:
it is used in two places, whats the better way to code this?

Reply:
add file which name as `consts.ts` or something like that in `assets` folder